### PR TITLE
Issue #367: Add replay ticks column to game responses

### LINKED ISSUE #367: Add column `replay_ticks` to `game` query response
A while ago I added the column `replay_ticks` to the `game_stats` table. Once the API returns this column in `data/game` queries, we can use it to display game durations more accurately in the vault.

### PR DESCRIPTION
**Pull request by Askaholic**
I'm assuming this is all that needs to be done, and Elide magic will make everything else work.

Closes #367.

### CODE REVIEW

File: src/main/java/com/faforever/api/data/domain/Game.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
   private int id;
   private OffsetDateTime startTime;
   private OffsetDateTime endTime;
+  private int replayTicks;
```
Comment (Brutus5000) [inline src/main/java/com/faforever/api/data/domain/Game.java]: As it is nullable in the db it should be `Integer`

File: src/main/java/com/faforever/api/data/domain/Game.java [hunk 2]
```diff
@@ -60,6 +61,11 @@ public OffsetDateTime getStartTime() {
     return startTime;
   }
 
+  @Column(name = "replay_ticks")
+  public int getReplayTicks() {
```
Comment (Brutus5000) [inline src/main/java/com/faforever/api/data/domain/Game.java]: As it is nullable in the db it should be `Integer`

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
I'm not sure what is wrong with Travis CI. Are you sure your PR is based on the latest develop?
They might have changed something again -.-

**Comment by Askaholic** [comment]
Yea, I just pulled to make sure. Latest commit is 3801f1 'Code cleanup related to clans'.

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**